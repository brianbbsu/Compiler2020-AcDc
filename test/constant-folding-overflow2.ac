i a
a = 12345678 * 98765432
p a
