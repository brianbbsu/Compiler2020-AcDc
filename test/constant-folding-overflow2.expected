1219326221002896
